ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -1312,17 +1312,6 @@ periodics:
     job-release: "4.10"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-release-master-ci-4.10-e2e-aws-upgrade-single-node
-  reporter_config:
```

Comment:
Why are yo removing the slack reporting for single node ci? This seems unrelated to openstack

Reply:
weird. I just ran `make update`.

Let me try again.